src/components/CommentsList.jsx

```diff
@@ -14,13 +15,14 @@ export default function CommentsList() {
       setComments(comments);
       setIsLoading(false);
     });
-  }, []);
+  });
```

Comment:
I think you're missing something important here...
![image](https://github.com/user-attachments/assets/3e3d0a09-b227-4459-ba2e-7d3632a87c8d)

Reply:
cheers for spotting :)